backend/api/btb/api/schema/types/util.py

```diff
@@ -0,0 +1,45 @@
+import six
+from graphene import Scalar
+from graphql.language.ast import StringValue
+
+__limited_stringtype_cache = {}
+
+def LimitedString(maxLength, **kwargs):
+    if maxLength < 1:
```

Comment:
Just an idea: test here if `maxLength` is from enum `InputLengths`

Reply:
Hmm I don't know if we need this artificial limit here